packages/discord.js/src/structures/Message.js

```diff
@@ -644,7 +644,7 @@ class Message extends Base {
    * @property {MessageAttachment[]} [attachments] An array of attachments to keep,
    * all attachments will be kept if omitted
    * @property {FileOptions[]|BufferResolvable[]|MessageAttachment[]} [files] Files to add to the message
-   * @property {MessageActionRow[]|MessageActionRowOptions[]} [components]
+   * @property {ActionRow[]|MessageActionRowOptions[]} [components]
```

Comment:
```suggestion
   * @property {ActionRow[]|ActionRowOptions[]} [components]
```

ref: https://github.com/discordjs/discord.js/blob/34120bba971132077d9140b588eeea7a8b4c889d/packages/discord.js/typings/index.d.ts#L4529